app/src/main/java/ch/epfl/polybazaar/listing/Listing.java

```diff
@@ -35,7 +36,7 @@
      * @param userEmail
      * @param stringImage String format : you can use convertFileToString or convertStringToBitmap to convert into String
      */
-    public Listing(String title, String description, String price, String userEmail, String stringImage) {
+    public Listing(String title, String description, String price, String userEmail, String stringImage, String category) {
```

Comment:
Why don't you use constructor overloading?

Reply:
Because I think every Listing has to have a specified category